README.md

```diff
@@ -114,12 +117,76 @@ in realtime: they both erase the file with their data.
 
 So make sure you're modifying it from one location at a time with the latest up-to-date Todo.txt file.
 
+## Supported auth backends
+
+| Name | Configuration value | Additional PyPI dependencies |
+|------|---------------------|------------------------------|
+| Predefined users | `DictAuth` |  |
+| WebDAV auth | `WebDavAuth` | `webdavclient3` |
+
+The `DictAuth` uses the `USERS` dict from the config as user database. With this nothing fancy happens
+at all. The `WebDavAuth` has no local user database. It forwards the login data from the user to the
+configured webdav server and tries to log in on his behalf. If this is successful, access will be granted.
+
 ## Supported storage backends
 
 | Name | Configuration value | Additional PyPI dependencies |
 |------|---------------------|------------------------------|
 | Local file system | `FileSystem` |  |
 | [Dropbox](https://www.dropbox.com/) | `Dropbox` | `dropbox` |
+| WebDAV | `WebDav` | `webdavclient3` |
```

Comment:
Can you add a link to the official WebDAV website or even its english Wikipedia page?

Reply:
I set the link to the server-support list at wikipedia.